assets/properties-panel.css

```diff
@@ -1112,4 +1121,57 @@ textarea.bio-properties-panel-input {
 .bio-properties-panel-feel-checkbox .bio-properties-panel-feel-entry:not(.feel-active) .bio-properties-panel-feel-container,
 .bio-properties-panel-feel-toggle-switch .bio-properties-panel-feel-entry:not(.feel-active) .bio-properties-panel-feel-container {
   margin-left: auto;
+}
+
+.bio-properties-panel-entry:has(.bio-properties-panel-tooltip-wrapper) .bio-properties-panel-label {
+  text-decoration: underline;
+  text-decoration-style: dotted;
+  text-underline-offset: 2px;
+}
+
+.bio-properties-panel-entry:has(.bio-properties-panel-tooltip) {
```

Comment:
I love `:has`, but it's not supported in Firefox: https://caniuse.com/?search=%3Ahas

Reply:
Forgot about that 🙈